portal/src/texts/profile/picture.md

```diff
@@ -1,5 +1,288 @@
+	•	Skade
+	•	Vi tar biler rundt og i bilen uten at det ser ut som en bilreklame
+	•	Vi bruker typiske turistsituasjoner for reiseforsikring
+	•	Dyreforsikringsbilder kan være dyr alene eller med folk
+
+## Lys og farger
+Vi skaper en inviterende og virkelighetsnær atmosfære ved å bruke dagslys eller naturlig lyssetting og fargekorrigering.
+
+[bilde: prakteksemplar]
+
+Vi unngår skarpt lys og harde skygger, spesielt i ansikter.
+
+[bilde: skygger i ansikt, bilde der hovedmotivet forsvinner i skygge]
+
+Unngå hvit bakgrunn som skaper høy kontrast og skarpe kanter. Dette oppstår ofte der det er hvitmalte vegger eller ute med lys himmel.
+
+[bilde: bilde med overeksponert himmel + riktig eksponering eller annet vær + innebilde med ulik veggfarge]
+
+Vi bruker ikke effekter, filtre eller overflødig bruk av refleksjoner (lens flare) på bildene våre. Alle bildene i Fremtinds bildedatabase er redigert og fargekorrigert så de oppfyller kravene i bildestilen og oppleves like.
+
+[bilde: filter og lens flare]
+
+Fremtinds fargepallett er jordnær og  bildene kan gjerne tar opp Fremtinds støttefarger.
+
+	•	Lys og farger skal oppleves som naturlig
+	•	Unngå harde skygger
+	•	Unngå filtre og refleksjoner
+	•	Ta opp farger fra  Fremtinds fargepalett
+
+## Komposisjon
+Bildestilen der vi står på toppen av tinden og ser ned er karakteristisk for Fremtind. Her kan vi fly en kilometer over bakken og få oversikt over storslått natur, byer og tettsteder eller vi kan zoome inn og sveve noen meter over menneskene og få oversikt over en situasjon som utspiller seg. Til sist kan vi zoome helt inn, ca. en halvmeter over bildeflaten og få med detaljene på et middagsbord, en skrivepult eller noe som rører seg i naturen. Vi passer på at vi ser vinkelrett ned på bakken.
+
+[bilde: 3 x zoom]
+
+Når vi står på bakkeplan sammen med menneskene passer vi på at vi står vinkelrett mot bakgrunnen og danner et ettpunktsperspektiv. Horisontlinjen skal være vannrett, mens eventuelle mennesker eller gjenstander kan gjerne bryte opp det stramme uttrykket. (Wes Andreson og Stanley Cubrick er kjent for sin ekstreme bruk av ettpunktsperspektivet). 
+
+[bilde: Verksteddama og portrett fra Brandpad]
+
+Bildene skal brukes på responsive nettsider, og både format og størrelse kan bli endret, samtidig som andre elementer kan endre plassering i forhold til bildet. Med godt med rom rundt hovedmotivet i bildet kan man enklere plassere designelementer over bildet og man har et sikkerhetsrom rundt hovedmotivet slik at hovedmotivet ikke blir beskåret ved skalering.
+
+[animasjon: Vise samme bilde på flere flater med  og uten luft]
+
+Bildets format skal være rektangulært. Det skal ikke beskjæres så det blir for smalt. Minimums høyde- og breddeforhold er 2:35:1 (cinemascope) Vi legger aldri masker over bildene slik at de har andre geometriske former og vi fyller ikke objekter eller typografi med fotofyll. Hovedmotivet skal ikke beskjæres.
+
```

Comment:
Trenger vi å poengtere rektangulære bilder med tanke på responsivitet. I overgang fra fullskjerm desktop (liggende format) til mobil (stående format) er det en sjanse for at at kvadratisk format dukker opp innimellom. Vi har ikke kontroll på hvordan brukeren skalerer browservinduet. 
Usikker på om vi kan definere cinemascope format som smaleste alternativ.
(Steinop)